packages/table/preview/index.tsx

```diff
@@ -452,6 +457,15 @@ class MutableTable extends React.Component<{}, IMutableTableState> {
     }
     // tslint:enable no-console
```

Comment:
Still need a lint disabler somewhere due to [L455](https://github.com/palantir/blueprint/pull/1210/files/3a82e82678bccd36f56ca2172f8b07885d6f4c36#diff-f05438f506492986496f2e9127652e8dR455), but I'll make it a single `tslint:disable-next-line no-console`